display_api/display.lua

```diff
@@ -269,13 +278,21 @@ end
 function display_api.register_display_entity(entity_name)
 	if not minetest.registered_entities[entity_name] then
 		minetest.register_entity(':'..entity_name, {
-			collisionbox = { 0, 0, 0, 0, 0, 0 },
-			visual = "upright_sprite",
-			textures = {},
+			initial_properties = {
+				collisionbox = {0, 0, 0, 0, 0, 0},
+				visual = "upright_sprite",
+				textures = {},
+				physical = false,
```

Comment:
is this needed? default according to the docs

Reply:
hmm, not sure why github turned one line into 4, refering to `physical = false,`